integration_tests/build.gradle.kts

```diff
@@ -44,4 +48,25 @@ tasks.test {
     systemProperty("working-directory", System.getProperty("working-directory"))
     systemProperty("output-pattern", System.getProperty("output-pattern"))
     systemProperty("expected-output-code", System.getProperty("expected-output-code"))
+    filter {
+        excludeTestsMatching("*IT")
+    }
+}
+
+tasks.register<Test>("integrationTests") {
+    group = "Verification"
+    description = "Runs flank integration tests"
+    filter {
+        includeTestsMatching("*IT")
+    }
+    testLogging {
+        events("skipped", "failed")
+        exceptionFormat = TestExceptionFormat.FULL
+    }
+    maxParallelForks = 4
```

Comment:
reason for it to be 4?

Reply:
because I forgot to make it dependent on available processors 😅 
I will change it to 
```
maxParallelForks = Runtime.getRuntime().availableProcessors() / 2
```

Reply:
could greatly speed it all up